src/geom/bounding_box.C

```diff
@@ -62,6 +62,62 @@ bool BoundingBox::contains_point (const Point & p) const
 }
 
 
+
+Real BoundingBox::max_size() const
+{
+  Real size = 0;
+  for (unsigned int d = 0; d != LIBMESH_DIM; ++d)
```

Comment:
In the future, after #3895 goes in, we could use `make_range` with `libmesh_dim` here